cranelift/filetests/filetests/isa/x64/b1.clif

```diff
@@ -1,8 +1,8 @@
 test compile precise-output
```

Comment:
we can probably call this file something other than `b1.clif` now -- maybe `conditional_values.clif` or something?

Reply:
Good point, the existing name isn't really all that meaningful going forward :)